ifrs17-template/Initialization/InitSystemorphRefDataToMemory.ipynb

```diff
@@ -31,8 +31,7 @@
         {
             "cell_type": "code",
             "source": [
-                "#!import \"../Constants/CalculationEngine\"",
-                "\n#!eval calculationEngine"
+                "#!import \"../Constants/Sharepoint\""
```

Comment:
To me is strange that in this init notebook you reference a NB related to Sharepoint, just from the structure. Say you don't have SP and see this in the code, it looks strange.

Reply:
Right, I moved now the second part of the file into the places where we perform exports, and reverted the references to the old ones.